kit/media/src/UIVideo/types.ts

```diff
@@ -28,17 +38,24 @@ export type UIVideoProps = {
      * Set the aspectRatio of the player
      */
     aspectRatio?: number;
+    /**
+     * By analogy with the Image#resizeMode
+     * `contain` by default
+     */
+    resizeMode?: UIVideoResizeMode;
     /**
      * Callback function that is called when the media is loaded and ready to play
      */
     onLoad?: () => void;
     /**
      * Callback function that is called when video cannot be loaded
      */
-    onError?: () => void;
+    onError?: (error: any) => void;
```

Comment:
Here is `any` because type of error in `react-player` is `any` and IDK it's real format.

Reply:
Hm... OK.